src/index.html

```diff
@@ -9,6 +9,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <div class="flex_container">
+        <a href="#home" class="logo_link">
+          <img
+            src="images/logo.png"
+            alt="logo"
+            class="logo flex_item"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">Samsung</a>
+            </li>
```

Comment:
```suggestion
            <li class="nav__item">
              <a href="#apple" class="nav__link is-active">Apple</a>
            </li>

            <li class="nav__item">
              <a href="#samsung" class="nav__link">Samsung</a>
            </li>
```

Add blank line between multiple sibling blocks of html

FIx all such cases